Sources/AWSLambdaRuntimeCore/Lambda+String.swift

```diff
@@ -16,7 +16,7 @@ import NIOCore
 extension EventLoopLambdaHandler where Event == String {
     /// Implementation of a `ByteBuffer` to `String` decoding.
     @inlinable
-    public func decode(buffer: ByteBuffer) throws -> String {
+    public func decode(buffer: ByteBuffer) throws -> Event {
```

Comment:
These changes help to solve ambiguities in the swift compiler across default implementations of `EventLoopLambdaHandler` when importing `AWSLambdaRuntime` or running tests for the package in Xcode